### DISCUSSION TITLE
device rebooting problem

### DISCUSSION THREAD

**Discussion by Bertan0** [jomjol/AI-on-the-edge-device#1452] [Q&A]
my device is constantly rebooting. How can I solve this error

![image](https://user-images.githubusercontent.com/116979648/205133351-17449aa4-b5ea-4068-9127-f3eed50e667e.png)

**Comment by caco3** [comment]
The log would have provided you the answer:
"Not enough PSRAM available", "Firmware can not start"

So that means your hardware is not supported. You only have 2 MBytes of PSRAM but 4 MBytes are required!
